src/pages/storage/forms/StoragePoolFormMain.tsx

```diff
@@ -112,11 +126,8 @@ const StoragePoolFormMain: FC<Props> = ({ formik }) => {
             required
             disabled={!formik.values.isCreating}
           />
-          {!isCephDriver &&
-            !isCephFSDriver &&
-            !isDirDriver &&
-            !isPureDriver &&
-            !isPowerFlexDriver && (
+          {hasSize &&
+            (!canClusterSize || !isClusteredServer(settings) ? (
```

Comment:
After removing canClusterSize, we should revert the conditional to avoid the negation here and make this easier to read. Positive (non-negated) boolean usage is easier to read in general.